### DISCUSSION TITLE
Lock windows and automatically change a window region between genomes

### DISCUSSION THREAD

**Discussion by marta-coronado** [GMOD/jbrowse-components#3454] [Ideas]
Hi,

we have been working on a genomic browser, DrosOmics, that compiles 52 _D. melanogaster_ natural population genome samples: http://gonzalezlab.eu/drosomics. We also have a preprint: https://www.biorxiv.org/content/10.1101/2022.07.22.501088v2. Thanks to JBrowse 2, DrosOmics is the first population genetics browser that allows for the simultaneous navigation of multiple genomes.

DrosOmics has been under peer-review and we have received a comment that we want to translate as a request of two new features. First, the ability to "lock" a selected window and navigate upstream and downstream in all genome views simultaneously with a single scroll. Currently, this must be done manually for each genome view, which can be time-consuming and laborious, especially when dealing with up to 52 genomes. Additionally, the second feature would be the ability to search for the same region in different genomes at once, for example, to display the region of a particular gene in the genomes currently open. At the moment, this requires searching individually in each genome.

We believe that these two features would improve the user experience of JBrowse 2 and DrosOmics. Do you think they can be implemented in future updates of JBrowse 2?

I also want to acknowledge that [one of the features I requested some time ago](https://github.com/GMOD/jbrowse-components/discussions/2514) was successfully implemented in the newer versions of JBrowse 2. 

Thank you so much!

Marta

**Comment by cmdcolin** [comment]
this is great to hear Marta! I will try to write a more thorough reply as there are many interesting aspects to this, but just briefly, this could maybe relate to our "multi-way synteny" goals. our synteny view is very pairwise at the moment, basically showing only two genomes at a time, but adding more with linked scrolling would be very powerful

**Reply by marta-coronado** [reply]
Great Colin! Do you think that the linked-scrolling can be implemented?

Also, regarding the other feature: the ability to look for the same e.g. gene in different genomes assemblies, is that feasible?

I wait for the thorough response. Thank you very much!
